PmipRunner.cs

```diff
@@ -59,17 +59,19 @@ private bool TryGetPmipFunction(out PmipFunctionDataItem pmipFunction)
 			if (pmipFunction != null)
 				return true;
 
-			foreach (var module in Modules)
+			foreach (var module in this._frame.RuntimeInstance.GetModuleInstances())
 			{
-				var definition = $"{{,,{module}}}mono_pmip";
-
-				PmipFunctionDataItem item = null;
-				if (!EvaluateExpression(definition, r => item = new PmipFunctionDataItem {PmipFunction = definition}))
-					continue;
-
-				pmipFunction = item;
-				_stackContext.SetDataItem(DkmDataCreationDisposition.CreateAlways, item);
-				return true;
+				if (module is DkmNativeModuleInstance)
+				{
+					DkmNativeInstructionAddress address = ((DkmNativeModuleInstance) module).FindExportName("mono_pmip", true);
+					if (address != null)
```

Comment:
Same here for var and nesting. Also please specify the argument name for the literal true.

```csharp
var address = nativeModule.FindExportName("mono_pmip", whatDoesThisDo: true)
if (address == null)
    continue;
```